src/NuGet.Core/NuGet.Build.Tasks/Strings.resx

```diff
@@ -200,4 +200,8 @@
     <comment>0 - the path to the EXE being launched
 1 - The command-line arguments</comment>
   </data>
-</root>
+  <data name="Log_RestoreNoCacheInformation" xml:space="preserve">
+    <value>NoCache is deprecated in favor of the appropriately named NoHttpCache.</value>
```

Comment:
This contains `NoCache`, but the property name is `RestoreNoCache`. We should use the full property names.

Reply:
Good catch, thank you!